nowcasting_dataset/config/model.py

```diff
@@ -327,6 +353,7 @@ def set_base_path(self, base_path: str):
             "pv.pv_filename",
             "pv.pv_metadata_filename",
             "satellite.satellite_zarr_path",
+            "satellite_hrv.satellite_zarr_path",
```

Comment:
I may be getting confused, but should this be:

```suggestion
            "hrvsatellite.hrvsatellite_zarr_path",
```

Reply:
Yeah, you are right! I got a bit confused